librz/analysis/jmptbl.c

```diff
@@ -255,8 +245,7 @@ static bool detect_casenum_shift(RzAnalysisOp *op, RzRegItem **cmp_reg, st64 *st
 	return false;
 }
 
-// TODO: RENAME
-RZ_API bool try_get_delta_jmptbl_info(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 jmp_addr, ut64 lea_addr, ut64 *table_size, ut64 *default_case, st64 *start_casenum_shift) {
+RZ_API bool rz_analysis_get_delta_jmptbl_info(RzAnalysis *analysis, RzAnalysisFunction *fcn, ut64 jmp_addr, ut64 lea_addr, ut64 *table_size, ut64 *default_case, st64 *start_casenum_shift) {
```

Comment:
I wonder if it makes sense to merge some of those parameters in some kind of context like you did in other cases.
Also, doxygeb,